README.md

```diff
@@ -32,11 +32,11 @@
 ## 🛠 Built With <a name="built-with"></a>
 
```

Comment:
[OPTIONAL]
You may remove the live version link as it doesn't lead us anywhere. 👍

Reply:
Thanks, I will fix this